### DISCUSSION TITLE
Welcome to NoFWL Discussions!

### DISCUSSION THREAD

**Discussion by lencx** [lencx/nofwl#22] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Lling00** [comment]
安装后，打开窗口没有gpt出现，点关闭时，软件很难关闭。
![image](https://user-images.githubusercontent.com/48402714/234209970-08866c0b-89d8-4159-a125-6f63955f121d.png)
